docs/design_proposals/pixi_global_manifest.md

```diff
@@ -31,6 +31,9 @@ Among other things it will be written in the TOML format, be named `pixi-global.
 The motivation for the location is discussed [further below](#multiple-manifests)
 
 ```toml title="pixi-global.toml"
+[envs.python]
+channels = ["https://fast.prefix.dev/conda-forge"] # optional, defaults to "conda-forge"
```

Comment:
I think this should default to your global config.

Reply:
How about `optional, defaults to global config, which defaults to "conda-forge"`?

Reply:
Perfect!